src/store/Modules/Theme.ts

```diff
@@ -123,9 +124,10 @@ export default class Connections extends VuexModule {
       this.context.commit("updateGlobalTheme", { data, themeIdx })
     } catch (err) {
       console.log(err)
+      this.context.commit("updateGlobalTheme", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.